e2e/sidecars/sidecars_test.go

```diff
@@ -54,7 +55,9 @@ sidecars:
     image: %s    # Image URL for sidecar container.
     variables:
       NGINX_PORT: %s
+    env_file: ./magic.env
 logging:
+  env_file: ./magic.env
```

Comment:
Do we need to validate the env files are indeed added to the service?

Reply:
yeah, good call

Reply:
Done. I settled for writing some logic that checks that there is in fact an env file included with each container. Taken along with the fact that the service won't run if the permissions are wrong or the env file can't be found or uploaded, this should be sufficient.